plugins/jabra-file/fu-jabra-file.rs

```diff
@@ -0,0 +1,23 @@
+// Copyright 2024 Richard Hughes <richard@hughsie.com>
+// SPDX-License-Identifier: LGPL-2.1-or-later
+
+
+#[repr(u8)]
+enum FuJabraFileCmdCmd {
+    Unknown02 = 0x02,
+    Unknown03 = 0x03,
+    Unknown07 = 0x07,
+    Unknown26 = 0x26,
```

Comment:
@gdpcastro can you rename these to reality please? I think having these as a enum type will be helpful for debugging -- e.g. make the compiler check the type is correct for each call.